GrafCalculator/app/src/main/java/dev/software/bing/NotesAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package dev.software.bing
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import dev.software.bing.databinding.ItemNoteBinding
+
+class NotesAdapter(private val items: ArrayList<String>) : RecyclerView.Adapter<NoteViewHolder>() {
```

Comment:
лучше передать интерфейс и не завязываться на конкретной реализации
```suggestion
class NotesAdapter(private val items: List<String>) : RecyclerView.Adapter<NoteViewHolder>() {
```